dist/bin/kctf-cluster

```diff
@@ -263,8 +265,19 @@ EMAIL_ADDRESS=${EMAIL_ADDRESS}
 EOF
   if [[ $? -ne 0 ]]; then return 1; fi
 
+  set_lastconfig_link || return
+
   case "${CLUSTER_TYPE}" in
     gce)
+      if ! command -v gcloud >/dev/null 2>&1; then
+        if [[ "${START_CLUSTER}" == "1" ]]; then
+          _kctf_log_err "Can't start cluster, configuration created only locally. Gcloud not installed."
+          return 1
+        else
+          _kctf_log_warn "Configuration created only locally. Gcloud not installed."
```

Comment:
Is the GCloud relevant to the user? Should we warn when starting instead only?

Reply:
I believe it's something we should warn about. It's a weird case if you create a gce config without having gcloud installed.